spec/pubid_iso/identifier_spec.rb

```diff
@@ -517,6 +517,18 @@ module Pubid::Iso
       it_behaves_like "converts pubid to pubid"
     end
 
+    context "ISO/IEC/IEEE DTS 17301-1-1:2016(en)" do
+      let(:pubid) { "ISO/IEC/IEEE DTS 17301-1-1:2016(en)" }
+
+      it_behaves_like "converts pubid to pubid"
+    end
+
+    context "ISO/IEC/IEEE FDPAS 17301-1-1:2016(en)" do
+      let(:pubid) { "ISO/IEC/IEEE FDPAS 17301-1-1:2016(en)" }
```

Comment:
Can we change "FDPAS" to "FDTR" here? Technically FDPAS doesn't exist because a PAS does not have a FDIS stage.